libvast/src/type.cpp

```diff
@@ -61,7 +61,7 @@ type& type::name(const std::string& x) & {
   return *this;
 }
 
-type&& type::name(const std::string& x) && {
+type type::name(const std::string& x) && {
```

Comment:
I think the rvalue return type was actually correct.

https://foonathan.net/2018/03/rvalue-references-api-guidelines/

Reply:
The very blog post you linked advises not to do so, unless you're moving a member variable. The change made by @tobim is correct. Returning by rvalue-reference may lead to dangling references.